packages/components/src/components/event/index.ts

```diff
@@ -0,0 +1,14 @@
+export { default as EventCard } from './eventCard/EventCard';
+export { default as LargeEventCard } from './eventCard/LargeEventCard';
+export { default as EventKeywords } from './eventKeywords/EventKeywords';
+export { default as EventHero } from './eventHero/EventHero';
+export { default as EventClosedHero } from './eventHeroClosed/EventClosedHero';
+export { default as EventContent } from './eventContent/EventContent';
+export { default as EventInfo } from './eventInfo/EventInfo';
+export { SubEvents, SuperEvent } from './eventInfo/EventsHierarchy';
+export { default as OrganizationInfo } from './eventInfo/OrganizationInfo';
+export { default as OtherEventTimes } from './eventInfo/OtherEventTimes';
+export { default as EventPageMeta } from './eventPageMeta/EventPageMeta';
+export { default as SimilarEvents } from './similarEvents/SimilarEvents';
+export { default as EventDetails } from './eventDetails/EventDetails';
+export type { EventDetailsProps } from './eventDetails/EventDetails';
```

Comment:
The EventDetails is exported twice with the same name. There is a type named EventDetails and now there is this component. It can be seen with a linter and verified with `yarn typecheck`. The symptom is that the application does not build succesfully, which can also be seen from the ci-pipeline result.